src/content/1.7/development/webservice/tutorials/4-advanced-use/additional-list-parameters.md

```diff
@@ -0,0 +1,102 @@
+    'display'  => 'full'
+);{{< /code >}} |
+| Include only the *ID* from the `carriers` resource | `/api/carriers/?display=[id]` | {{< code php >}}$opt = array(
+    'resource' => 'carriers',
+    'display'  => '[id]'
+);{{< /code >}} |
+| Only include the *name* and *value* fields from the `configurations` resource | `/api/configurations/?display=[name,value]` | {{< code php >}}$opt = array(
+    'resource' => 'configurations',
+    'display'  => '[name,value]'
+);{{< /code >}} |
+
+## Filter parameter
+
+You can filter the expected resulted with the `filter` parameter
+
+| Key               | Value                               | Result                                                              |
+|-------------------|-------------------------------------|---------------------------------------------------------------------|
+| **filter[field]** | {{< code >}}[1|5]{{< /code >}} | `OR` operator: list of possible values                              |
+|                   | {{< code >}}[1,10]{{< /code >}}     | `Interval` operator: define interval of possible values             |
+|                   | {{< code >}}[John]{{< /code >}}     | `Literal` value (not case sensitive)                                |
+|                   | {{< code >}}[Jo]%{{< /code >}}      | `Begin` operator: fields begins with the value (not case sensitive) |
+|                   | {{< code >}}%[hn]{{< /code >}}      | `End` operator: fields ends with the value (not case sensitive)     |
+|                   | {{< code >}}%[oh]%{{< /code >}}     | `Contains` operator: fields contains the value (not case sensitive) |
+
+| Result | API call | PHP Webservice lib options |
+|--------|----------|----------------------------|
+| Only the `customers` whose ids are 1 or 5 | {{< code >}}/api/customers/?filter[id]=[1|5]{{< /code >}} | {{< code php >}}$opt = array(
+    'resource' => 'customers',
+    'filter[id]'  => '[1|5]'
+);{{< /code >}} |
+| Only the `customers` whose ids are between 1 and 10 | {{< code >}}/api/customers/?filter[id]=[1,10]{{< /code >}} | {{< code php >}}$opt = array(
+    'resource' => 'customers',
+    'filter[id]'  => '[1,10]'
+);{{< /code >}} |
+| Only the `customers` whose first name is "John" | {{< code >}}/api/customers/?filter[firstname]=[John]{{< /code >}} | {{< code php >}}$opt = array(
+    'resource' => 'customers',
+    'filter[firstname]' => '[John]',
+);{{< /code >}} |
+| Only the `manufacturers` whose name begins with "Appl" | {{< code >}}/api/manufacturers/?filter[firstname]=[John]{{< /code >}} | {{< code php >}}$opt = array(
+    'resource' => 'manufacturers',
+    'filter[name]' => '[appl]%',
+);{{< /code >}} |
+
+## Sort parameter
+
+You can sort the expected resulted with the `sort` parameter
+
+| Key      | Value | Result |
+|----------|-------|--------|
+| **sort** | {{< code >}}[{fieldname}_{ASC|DESC}]{{< /code >}} | The `sort` value is composed of a field name and the expected order separated by a `_` |
```

Comment:
```suggestion
| **sort** | `[{fieldname}_{ASC|DESC}]` | The `sort` value is composed of a field name and the expected order separated by a `_` |
```

Reply:
This one is necessary because of the `|`